packages/google_maps_flutter/google_maps_flutter_ios/ios/Classes/FLTGoogleMapTileOverlayController.m

```diff
@@ -83,6 +79,11 @@ - (void)interpretTileOverlayOptions:(NSDictionary *)data {
   if (tileSize) {
     [self setTileSize:tileSize.integerValue];
   }
+
+  NSNumber *visible = FGMGetValueOrNilFromDict(data, @"visible");
+  if (visible) {
+    [self setVisible:visible.boolValue];
+  }
```

Comment:
This doesn't appear to be necessary for your change. Please revert to minimise diffs.